freerdp/freerdp.nuspec

```diff
@@ -3,10 +3,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>freerdp</id>
-    <version>1.1</version>
-    <title>FreeRDP (Portable)</title>
+    <version>1.1.20230113</version>
```

Comment:
@Roemer since package versions are immutable, meaning once they are approved they can't be changed, the only way to make these changes, and have them appear on the site, is to push a new package fix notation version.  This is a well established route when a change to the package, but not the underlying application, is required.